lib/PrintAsClang/PrintClangFunction.cpp

```diff
@@ -713,6 +713,18 @@ ClangRepresentation DeclAndTypeClangFunctionPrinter::printFunctionSignature(
     if (kind == FunctionSignatureKind::CxxInlineThunk)
       ClangSyntaxPrinter(os).printGenericSignature(Signature);
   }
+  if (const auto *enumDecl = FD->getDeclContext()->getSelfEnumDecl())
+  {
+    // We cannot emit functions with the same name as an enum case yet, the resulting header
+    // does not compiler.
+    // FIXME: either do not emit cases as inline members, or rename the cases or the
+    //        colliding functions.
+    for (const auto *enumElement : enumDecl->getAllElements()) {
```

Comment:
I am not sure how performance sensitive is this code path. Note that, this could potentially be "quadratic": `num(elements)*num(methods)`. An asymptotically better behaving solution might do some kinds of lookup by name.